### PR TITLE
Prepare for release 1.0.0 "Naked Chicken"

### PR DESCRIPTION
**Pull request by apeltzer**
This will ultimately be the first release of nf-core/openmspeptidequant, "Naked Chicken". 

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/openmspeptidequant branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/openmspeptidequant)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/openmspeptidequant/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by ewels** [comment]
_Naked Chicken_ - really?? 😆

**Comment by ewels** [comment]
Few minor things:

* `main.nf` is missing ASCII art for the help text
* `nextflow.config` has the `standard` profile twice
* Most of the configs in `base.config` could be grouped into a couple of `withLabel` statements
* Would be nice to harmonise the profiles a bit with the base template (add uppmax, add zurich to the template)
* Nextflow [throws a warning](https://travis-ci.com/nf-core/openmspeptidequant/jobs/159769237#L713):
  * ```
    WARN: The operator `first` is useless when applied to a value channel which returns a single value by definition -- check channel `id_files_merged_psm_perc_filtered`
    ```
*  Strange minor markdown rendering bug in docs:
![image](https://user-images.githubusercontent.com/465550/48785132-6e2de400-ece4-11e8-8b47-9e3ead85dbb7.png)
* `num_threads` is [documented](https://github.com/nf-core/openmspeptidequant/blob/master/docs/usage.md#--num_threads) (and [here](https://github.com/nf-core/openmspeptidequant/blob/063bfe80f82374606ba37aa7ba13c33bc8d9cd16/main.nf#L49)) and [in `nextflow.config`](https://github.com/nf-core/openmspeptidequant/blob/063bfe80f82374606ba37aa7ba13c33bc8d9cd16/nextflow.config#L23) but not used in `main.nf`
* [Docs](https://github.com/nf-core/openmspeptidequant/blob/master/docs/usage.md#-profile) don't describe several profiles

Generally looking really good though!

Phil

**Comment by apeltzer** [comment]
> _Naked Chicken_ - really?? 😆

Yes, @Leon-Bichmann had the idea, not my choice :-D

**Comment by apeltzer** [comment]
All points mentioned by @ewels addressed now:

- ASCII Art added
- got rid of the `first()` statement which was unnecessary
- Added documentation fixes for markdown rendering issues
- Added Docs for Profiles present atm (others will come via Sync) 
- Submitted separate PR to tools for having more profiles there (e.g. UZH, also introduced sorting there in the PR)
- Removed redundant resource statements in the `base.config`, these are now defaults 
  * Only resource requests bigger or smaller get their own labels now
- Got rid of `num_threads`which isn't used at all in the pipeline anymore

File: docs/usage.md [hunk 1]
```diff
@@ -15,7 +15,7 @@
     * [`--mzmls`](#--mzmls)
```
[the remark this answers is not in the document]
Comment (apeltzer) [reply docs/usage.md]: That would be something for an issue for the next release then...?

[the remark this answers is not in the document]
Comment (Leon-Bichmann) [reply docs/usage.md]: yes probably better to shift it to the next release

[the remark this answers is not in the document]
Comment (Leon-Bichmann) [reply docs/usage.md]: unless there is a very quick fix to it

[the remark this answers is not in the document]
Comment (apeltzer) [reply docs/usage.md]: I don't get it: Is this breaking something or is the pipeline usable this way? If its breaking something, we'd need to add it now :-D

[the remark this answers is not in the document]
Comment (Leon-Bichmann) [reply docs/usage.md]: Ah ok no its working like it is, just at the moment there is no way to process a large batch of replicate measurements.

As I see it one would have to submit each set of replicates separately.

[the remark this answers is not in the document]
Comment (apeltzer) [reply docs/usage.md]: Which is okay, but could be improved. So rather enhancement than something blocking a release right?

[the remark this answers is not in the document]
Comment (Leon-Bichmann) [reply docs/usage.md]: ok right, lets leave it for a later release

**Comment by apeltzer** [comment]
Complete rename solved :-)

**Comment by sven1103** [comment]
The openms version in the environment spec must be `openms=2.4.0`

**Comment by apeltzer** [comment]
Had to adapt the test-datasets first, now this works.

**Comment by Leon-Bichmann** [comment]
anything missing still for the chicken to leave the nest?

**Comment by apeltzer** [comment]
A review by @ewels for example ;-)

**Comment by apeltzer** [comment]
I think this is fine, will merge to dev atm

**Pull request merged**